notebooks/compare-pretraining-finetuning-performance.ipynb

```diff
@@ -0,0 +1,326 @@
+      "text/plain": [
+       "<Figure size 640x480 with 2 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "sns.heatmap(data, annot=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The End."
```

Comment:
Love this <3

Reply:
FINALLY someone that notices! ❤️ 

I'm trying to make it my signature! See e.g. [MOOD](https://github.com/valence-labs/mood-experiments/tree/main/notebooks). 😅